src/CssInliner.php

```diff
@@ -492,7 +492,9 @@ private function getCssFromAllStyleNodes(): string
 
         $css = '';
         foreach ($styleNodes as $styleNode) {
-            $css .= "\n\n" . $styleNode->nodeValue;
+            if ($styleNode->nodeValue !== null) {
```

Comment:
Let's make this more clear to the reader which type is expected (so the reader doesn't need to look up the non-null type of `DOMNode::nodeValue`):

```suggestion
            if (\is_string($styleNode->nodeValue)) {
```